### ISSUE TITLE
NameError: name 'unicode' is not defined

### ISSUE THREAD

**Issue by abadabanades** [hozn/stravalib#90]
After updating from `0.5.0` to `0.6.1` using `Python 3.4`, I get this error:

~~~
  File "/home/abadaba/dev/activities/lib/python3.4/site-packages/stravalib/model.py", line 233, in AthleteStats
    recent_ride_totals = EntityAttribute(ActivityTotals)  #: Recent totals for rides. (:class:`stravalib.model.ActivityTotals`)
  File "/home/abadaba/dev/activities/lib/python3.4/site-packages/stravalib/attributes.py", line 245, in __init__
    super(EntityAttribute, self).__init__(*args, **kwargs)
  File "/home/abadaba/dev/activities/lib/python3.4/site-packages/stravalib/attributes.py", line 37, in __init__
    self.type = type_
  File "/home/abadaba/dev/activities/lib/python3.4/site-packages/stravalib/attributes.py", line 258, in type
    if isinstance(v, (str, bytes, unicode)):
NameError: name 'unicode' is not defined
~~~

I found this in StackOverflow (http://stackoverflow.com/questions/19877306/nameerror-global-name-unicode-is-not-defined-in-python-3), so by simply removing the `unicode` check in line 258 (only checking for `(str, bytes)`) works and all my tests pass (not sure whether this will brake anything, though).

If you think it's ok, I'll do the PR

**Comment by hozn** [comment]
I can fix this. It should be using six. Not sure how that got through, must
have been an accidental merge from a recent PR.

**Comment by hozn** [comment]
This should be fixed now in just-pushed 0.6.2.  Unit tests pass on python 3 (3.5) and python 2, so I think we're good.  Sorry about that!

**Comment by abadabanades** [comment]
Thx! it works now!
